bigquery_etl/cli/query.py

```diff
@@ -835,11 +839,11 @@ def _run_query(
                     and destination_table is not None
                     and re.match(DESTINATION_TABLE_RE, destination_table)
                 ):
-                    destination_table = "{}:{}.{}".format(
+                    qualified_destination_table = "{}:{}.{}".format(
                         public_project_id, dataset_id, destination_table
                     )
                     query_arguments.append(
-                        "--destination_table={}".format(destination_table)
+                        "--destination_table={}".format(qualified_destination_table)
```

Comment:
Awkward moment because I think I suggested this naming change during the session haha but I think we should probably leave this block as-is at least in this PR since it's likely unrelated to backfills